com.vscodium.codium-insiders.yaml

```diff
@@ -0,0 +1,143 @@
+id: com.vscodium.codium-insiders
+sdk: org.freedesktop.Sdk
+runtime: org.freedesktop.Sdk
+runtime-version: '23.08'
+command: codium-insiders
+finish-args:
+  - --share=ipc
+  - --socket=x11
+  - --socket=wayland
+  - --socket=pulseaudio
+  - --socket=ssh-auth
+  - --share=network
+  - --device=all
+  - --filesystem=host
+  - --persist=.vscode-oss
+  - --allow=devel
+  - --talk-name=org.freedesktop.secrets
+  - --talk-name=org.freedesktop.Flatpak
+  - --env=XCURSOR_PATH=/run/host/user-share/icons:/run/host/share/icons
+  - --filesystem=xdg-run/gnupg:ro
+  - --filesystem=xdg-config/kdeglobals:ro
+  - --talk-name=com.canonical.AppMenu.Registrar
+  - --talk-name=com.canonical.AppMenu.Registrar.*
+  - --system-talk-name=org.freedesktop.login1
+separate-locales: false
+add-extensions:
+  com.visualstudio.code.tool:
+    directory: tools
+    subdirectories: true
+    version: '23.08'
+    add-ld-path: lib
+    no-autodownload: true
+cleanup:
+  - /include
+  - /lib/pkgconfig
+  - /share/gtk-doc
+  - '*.la'
+modules:
+  - shared-modules/libsecret/libsecret.json
+  - shared-modules/libusb/libusb.json
+  - name: codium-insiders
+    buildsystem: simple
+    build-commands:
+      - install -Dm644 codium-insiders.svg /app/share/icons/hicolor/scalable/apps/$FLATPAK_ID.svg
+      - |
+        for size in 16 32 48 64 128 256 512; do
+          rsvg-convert --width ${size} --keep-aspect-ratio --format png --output codium-insiders_${size}.png codium-insiders.svg || exit 1
+          install -Dm644 codium-insiders_${size}.png /app/share/icons/hicolor/${size}x${size}/apps/$FLATPAK_ID.png || exit 1
+        done
+      - install -Dm644 $FLATPAK_ID.metainfo.xml -t /app/share/metainfo
+      - ar x codium.deb
+      - tar xf data.tar.xz
+        # HACK: Use directory for IPC sockets that is shared between Flatpak instances
+        # Replace XDG_RUNTIME_DIR by XDG_RUNTIME_DIR/app/FLATPAK_ID
+      - find usr/share/codium-insiders -type f -print0 | xargs -0 sed -i "s/\bjoin([^)]*xdg_\?runtime_\?dir[^,)]*/\0,'app',process.env.FLATPAK_ID/gI"
+        # and disable integrity check
+      - sed -e 's/"checksums":/"_checksums":/' -i usr/share/codium-insiders/resources/app/product.json
+      - mv usr/share/codium-insiders /app/share
+        # use sed because because desktop-file-edit can't modify different group keys
+      - sed -e "s/^Icon=.*/Icon=$FLATPAK_ID/" -i usr/share/applications/*.desktop
+      - sed -e "s|/usr/share/codium-insiders/||g" -i usr/share/applications/*.desktop
```

Comment:
to modify the desktop file, use `desktop-file-edit`. Sed failure mode is inexistant.

Reply:
then make a patch. sed failure mode is unexistent which make anything hard to debug.

Reply:
> then make a patch. sed failure mode is unexistent which make anything hard to debug.

Done =)